documentation/docs/faq/liquidity-mining.md

```diff
@@ -100,59 +100,96 @@ The Hummingbot Miners app uses your Ethereum wallet address to:
+1. Go to [Binance](https://www.binance.com/) website.
+2. Complete the sign up process.
+3. Navigate to Spot wallet, click the **Deposit** button and choose **XZC** to display the Zcoin deposit address or choose **ETH** to display the Ethereum deposit address.
 
-!!! Note "Note"  
-    You may also refer to this step on [How to get your deposit address on Binance](https://academy.binance.com/tutorials/how-to-deposit#deposit).  
-    You can also check your ZCoin wallet on https://explorer.zcoin.io/  
-    Zcoin address start with:  "a..." or "Z..." or "3..." for multi-sig wallets.  
+Above steps are for new user, existing user can jump to step #3 after successful login.
 
-  
-#### Get Zcoin deposit address with mobile wallet  
-1. Download edge Wallet from Apple Store or Google Play.  
-2. Open the app and create an account - complete the sign up process.  
-3. Click the plus sign next to My **Wallets** to add a Zcoin wallet. Type **Zcoin** or select it from the list.  
-4. Choose your desired fiat pair, then name you wallet and confirm by clicking "Create Wallet".  
-5. Zcoin wallet will appear on the main screen, click Zcoin wallet then click **Request**. This generates an address with a correspoind QR code which you can copy and share to receive your coin.  
+!!! note
+    - You may also refer to this step on [How to get your deposit address on Binance](https://academy.binance.com/tutorials/how-to-deposit#deposit). 
+    - You can also check your ZCoin wallet on https://explorer.zcoin.io/
+    - `Zcoin address start with:  "a..." or "Z..." or "3..." for multi-sig wallets.`
+
+### Get Zcoin deposit address with mobile wallet
+
+1. Download edge Wallet from Apple Store or Google Play.
+2. Open the app and create an account - complete the sign up process.
+3. Click the plus sign next to My **Wallets** to add a Zcoin wallet. Type **Zcoin** or select it from the list.
+4. Choose your desired fiat pair, then name you wallet and confirm by clicking "Create Wallet".
+5. Zcoin wallet address will appear on the main screen, click Zcoin wallet then click **Request**. This generates an address with a correspoind QR code which you can copy and share to receive your coin.
 
 Above steps are for new user, existing user can jump to step #5 after successful login.
 
-!!! note "Note"
+!!! note
     Though above example uses edge wallet, Zcoin doesn't endorse one wallet in particular.
-    You may also refer to this step on [How to receive Zcoin with a mobile wallet](https://zcoin.io/how-to-receive-zcoin-with-a-mobile-wallet) 
-    And also this step on [Installation and and getting started with your Zcoin wallet](https://youtu.be/8-HKyM3QrJU)   
-      
+
+    - You may also refer to this step on [How to receive Zcoin with a mobile wallet](https://zcoin.io/how-to-receive-zcoin-with-a-mobile-wallet)
+    - And also this step on [Installation and and getting started with your Zcoin wallet](https://youtu.be/8-HKyM3QrJU)
+
+### Get NEM deposit address
+
```

Comment:
Add link to Binance deposit instructions. 

And add a note: "You can use a Binance.com NEM wallet address (and memo) to receive NEM directly into your Binance account".

Alternatively, you can create a separate NEM wallet: